cli/cmd/export/export.go

```diff
@@ -101,28 +101,28 @@ func collectObjectsFromNs(ctx *clicontext.CLIContext, ns string) ([]runtime.Obje
 		objects = append(objects, &services.Items[i])
 	}
 
-	for _, svc := range services.Items {
-		configs, err := configmapsFromService(ctx, svc)
+	for i := range services.Items {
+		configs, err := configmapsFromService(ctx, services.Items[i])
```

Comment:
This one isn't technically necessary since we aren't using it's pointer, but it future proofs us.